### ISSUE TITLE
Remove ExceptionPrettyPrinter from readme?

### ISSUE THREAD

**Issue by debfx** [jrobichaud/django-structlog#55]
When using the structlog config from the readme, exceptions are properly logged as JSON but also as plain text.
For development this might be helpful, but it was really surprising to me that enabling structlog still prints plain text. :grinning: 

After some debugging I figured out that `structlog.processors.ExceptionPrettyPrinter()` is responsible for this behavior.
It might be good idea to remove that processor from the readme or add a comment saying what it does.

**Comment by jrobichaud** [comment]
You found something interesting.


With `structlog.processors.ExceptionPrettyPrinter()`:
```json
{
  "request_id": "73fad830-dcf3-4e97-948d-3ae95f5b66b9",
  "user_id": null,
  "ip": "172.26.0.1",
  "code": 500,
  "request": "<WSGIRequest: POST '/success_task'>",
  "error": "Exception('Hey')",
  "error_traceback": "  File \"/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py\", line 113, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/usr/local/lib/python3.7/contextlib.py\", line 74, in inner\n    return func(*args, **kwds)\n  File \"/app/django_structlog_demo_project/home/views.py\", line 13, in enqueue_successful_task\n    raise Exception(\"Hey\")\n",
  "event": "request_failed",
  "timestamp": "2020-11-11T21:02:22.085776Z",
  "logger": "django_structlog.middlewares.request",
  "level": "error"
}
```

Without `structlog.processors.ExceptionPrettyPrinter()`:
```json
{
  "request_id": "9460c683-d9c7-40e0-8463-ee860f278c14",
  "user_id": null,
  "ip": "172.26.0.1",
  "code": 500,
  "request": "<WSGIRequest: POST '/success_task'>",
  "error": "Exception('foo')",
  "error_traceback": "  File \"/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py\", line 113, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/usr/local/lib/python3.7/contextlib.py\", line 74, in inner\n    return func(*args, **kwds)\n  File \"/app/django_structlog_demo_project/home/views.py\", line 13, in enqueue_successful_task\n    raise Exception(\"foo\")\n",
  "event": "request_failed",
  "timestamp": "2020-11-11T21:37:14.182907Z",
  "logger": "django_structlog.middlewares.request",
  "level": "error",
  "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py\", line 113, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/usr/local/lib/python3.7/contextlib.py\", line 74, in inner\n    return func(*args, **kwds)\n  File \"/app/django_structlog_demo_project/home/views.py\", line 13, in enqueue_successful_task\n    raise Exception(\"foo\")\nException: foo"
}
```

I will investigate this.

**Comment by jrobichaud** [comment]
I could remove the traceback I add myself to use the exception handled by the logger:

```json
{
  "request_id": "566bed59-e241-492d-9068-4458a7f5d132",
  "user_id": null,
  "ip": "172.26.0.1",
  "code": 500,
  "request": "<WSGIRequest: POST '/success_task'>",
  "error": "Exception('foo')",
  "event": "request_failed",
  "timestamp": "2020-11-11T21:52:20.730686Z",
  "logger": "django_structlog.middlewares.request",
  "level": "error",
  "exception": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/django/core/handlers/base.py\", line 113, in _get_response\n    response = wrapped_callback(request, *callback_args, **callback_kwargs)\n  File \"/usr/local/lib/python3.7/contextlib.py\", line 74, in inner\n    return func(*args, **kwds)\n  File \"/app/django_structlog_demo_project/home/views.py\", line 13, in enqueue_successful_task\n    raise Exception(\"foo\")\nException: foo"
}
```

This seems to be a small breaking change for people using the library.

**Comment by jrobichaud** [comment]
Fixed in 2.0.0
